install.sh

```diff
@@ -615,5 +685,76 @@ fi
 
 }
 
+function register_sandcats_name() {
+  # We allow environment variables to override some details of the
+  # Sandcats service, so that during development, we can test against
+  # a non-production Sandcats service.
+  SANDCATS_BASE_DOMAIN="${OVERRIDE_SANDCATS_BASE_DOMAIN:-sandcats.io}"
+  SANDCATS_API_BASE="${OVERRIDE_SANDCATS_API_BASE:-https://sandcats.io}"
+  SANDCATS_CURL_PARAMS="${OVERRIDE_SANDCATS_CURL_PARAMS:-}"
+
+  echo "Press enter to accept a random name, or choose your own (alphanumeric, max len 20)."
```

Comment:
I worry that someone might accidentally press enter, or press enter without thinking, and get a random name, which is not what they intended. Perhaps they should type "random" for a random name. I know your argument is that simply repeatedly pressing enter should produce a reasonable result, but IMO a randomly-generated name is probably _not_ what the vast majority of people actually want, and so this will probably just lead to confusion...